Directory.Build.props

```diff
@@ -7,7 +7,7 @@
         <TestLibrary>false</TestLibrary>
         <NitroxLibrary>false</NitroxLibrary>
         <UnityModLibrary>false</UnityModLibrary>
-        <BuildToolDir>$(SolutionDir)Nitrox.BuildTool\bin\</BuildToolDir>
+        <BuildToolDir>$(MSBuildThisFileDirectory)Nitrox.BuildTool\bin\</BuildToolDir>
         <BuildGenDir>$(BuildToolDir)generated_files\</BuildGenDir>
         <BuildGenDllDir>$(BuildGenDir)publicized_assemblies\</BuildGenDllDir>
         <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
This file still contains references to the "Nitrox.Analyzers" project that can be removed:
```csproj
<PropertyGroup Condition="'$(NitroxProject)' == 'true' and '$(MSBuildProjectName)' != 'Nitrox.Analyzers'">
```